vanilla/scss/styles.scss

```diff
@@ -70,3 +70,20 @@ li > p {
 .admonition-security-notice {
   padding-left: 1rem;
 }
+
+.headerlink {
+  display: none;
```

Comment:
There are some `.headerlink` styles (color) in line 14. I guess move it down here as well to keep them all in one place.

Also, while at it we could make it look less broken.
I'm not sure if that should use text color, it makes it really black, maybe we can use muted color for that `$colors--light-theme--text-muted`.
Also, it inherits styles directly from `a` which gives it underline on hover and purple visited color. We should override all of those. Maybe something like?

```
.headerlink:is(:link, :hover, :focus, :visited) {
    color: $colors--light-theme--text-muted;
    text-decoration: none;
}
```